### DISCUSSION TITLE
New mod : Sliderlock (reduce slider follow circle width)

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17336] [Ideas]
### Sliderlock
New mod that reduces the slider follow circle to the slider's width
The player can no longer escape out of the slider's borders

![image](https://user-images.githubusercontent.com/40531871/159102146-7df8c0cd-ae3f-4a93-81a4-63b0836c69af.png)

**Comment by frenzibyte** [comment]
Pleas separate each proposal to its own thread. Hard to track one that asks for three different things each at a different area.

**Reply by AuracleTech** [reply]
Wouldn't that be considered spam, opening tons of threads
Thought about it, just didn't want to spam threads

**Reply by frenzibyte** [reply]
It's not considered "spam" at all. In general, each feature request should have its own discussion thread.
